recipes/briefcase/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "briefcase" %}
+{% set version = "0.3.25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/briefcase-{{ version }}.tar.gz
+  sha256: 0d3ca50714da8bed30e80ea33f322e6b8be7e80a1fb651814d98561aa269de72
+
+build:
+  skip: true  # [py<310]
+  entry_points:
+    - briefcase = briefcase.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools ==80.9.0
+    - setuptools-scm ==9.2.0
+    - pip
+  run:
+    - python
+    - packaging >=24.2
+    - pip >=24.3
+    - setuptools >=60
+    - wheel >=0.37
+    - python-build >=0.10
+    - truststore >=0.10.1
+    - cookiecutter >=2.6.0,<3.0
+    - dmgbuild >=1.6.4,<2.0  # [osx]
+    - gitpython >=3.1,<4.0
+    - platformdirs >=2.6,<5.0
+    - psutil >=5.9,<8.0
+    - python-dateutil >=2.9.0.post0
+    - httpx >=0.20,<1.0
+    - rich >=12.6,<15.0
+    - tomli >=2.0,<3.0  # [py<=310]
```

Comment:
If we allow this to be installed in py>=310 we can make this noarch. It is preferred b/c we save on CI time. We can drop it when droppping py<=310.